Makefile

```diff
@@ -0,0 +1,27 @@
+# Installation directories.
+PREFIX ?= $(DESTDIR)/usr
+BINDIR ?= $(PREFIX)/bin
+MANDIR ?= $(PREFIX)/share/man
+UNITDIR ?= $(PREFIX)/lib/systemd/system
+DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
+PROFILEDIR ?= $(DESTDIR)/etc/profile.d
+
+.PHONY: test
+test:
+	./tests/run-tests.sh
+
+.PHONY: clean
+clean:
+	-rm -rf *~ \#* .#*
+
+.PHONY: install
+install:
+
+	install -D -m 755 docker-storage-setup.sh ${BINDIR}/docker-storage-setup
+	install -D -m 644 docker-storage-setup.service ${UNITDIR}/docker-storage-setup.service
+	install -D -m 644 docker-storage-setup.conf ${DSSLIBDIR}/docker-storage-setup
+	install -D -m 644 docker-storage-setup-override.conf ${SYSCONFDIR}/docker-storage-setup
+	install -D -m 755 libdss.sh ${DSSLIBDIR}/libdss.sh
+	install -D -m 755 dss-child-read-write.sh ${DSSLIBDIR}/dss-child-read-write
```

Comment:
Most likely not going to matter, since this is more about packaging in an RPM which is not going to preserve timestamps.  I don't think these are usually preserved in a Makefile.